pom.xml

```diff
@@ -44,7 +44,7 @@
     <dependency>
       <groupId>org.apache.tomcat.embed</groupId>
       <artifactId>tomcat-embed-jasper</artifactId>
-      <scope>provided</scope>
```

Comment:
this change is local for idea, fine working with provided scope, don't change on master